packages/foreman/rubygem-deep_cloneable/rubygem-deep_cloneable.spec

```
@@ -1,77 +1,78 @@
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkgname %{name}}
-
-%define gem_name deep_cloneable
+# template: default
+%global gem_name deep_cloneable
 
+Name: rubygem-%{gem_name}
+Version: 3.2.0
+Release: 1%{?dist}
 Summary: This gem gives every ActiveRecord::Base object the possibility to do a deep clone
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 3.0.0
-Release: 4%{?dist}
 Group: Development/Languages
 License: MIT
 URL: https://github.com/moiristo/deep_cloneable
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix}rubygem(activerecord) >= 3.1.0
-Requires: %{?scl_prefix}rubygem(activerecord) < 7
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-BuildRequires: %{?scl_prefix_ruby}ruby
+# start specfile generated dependencies
+Requires: ruby >= 1.9.3
+BuildRequires: ruby >= 1.9.3
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+# end specfile generated dependencies
 
 %description
 Extends the functionality of ActiveRecord::Base#dup to perform a deep clone
-that includes user specified associations.
+that includes user specified associations. .
```

This is an example of where the template is insufficient. I've opened https://github.com/moiristo/deep_cloneable/pull/142 but for now:
```suggestion
that includes user specified associations.
```